_vendor/github.com/inspec/inspec/docs-chef-io/content/inspec/resources/os.md

```diff
@@ -123,7 +123,7 @@ Use `os.family` to enable more granular testing of platforms, platform names, ar
 - `bsd` For platforms that are part of the Berkeley OS family `darwin`, `freebsd`, `netbsd`, and `openbsd`.
 - `debian`
 - `hpux`
-- `linux`. For platforms that are part of the Linux family `alpine`, `amazon`, `arch`, `coreos`, `exherbo`, `fedora`, `gentoo`, and `slackware`.
+`linux`. For platforms that are part of the Linux family `alpine`, `amazon`, `arch`, `coreos`, `exherbo`, `fedora`, `gentoo`, and `slackware`.
```

Comment:
This should get yanked too and done in `inspec/inspec`

Reply:
I clearly messed up reverting that. Let me fully back it out.